skfda/exploratory/visualization/_magnitude_shape_plot.py

```diff
@@ -109,7 +107,7 @@ class MagnitudeShapePlot(BasePlot):
         ...                [-0.5, -0.5, -0.5, -1, -1, -1]]
         >>> grid_points = [ 0., 2., 4., 6., 8., 10.]
         >>> fd = skfda.FDataGrid(data_matrix, grid_points)
-        >>> MagnitudeShapePlot(fd)
```

Comment:
Why didn't this work.

Reply:
it didnt work because know it was calling _repr_svg_() instead of __repr__()

Reply:
That should not be possible. Have you debugged it to know where `_repr_svg_` is called?

Reply:
my bad, it actually works gonna upload it